### ISSUE TITLE
heat.pl doesn't update table properly when saving heatmap images to database

### ISSUE THREAD

**Issue by Drek282** [Drek282/PsychoStats#84]
This issue goes back to the prehistoric roots of PsychoStats, but it's probably time that a serious look was taken at it, and it was fixed.

**Comment by Solomenka** [comment]
Fckn heatmaps makes me crazy. Still doesn't work.

**Comment by Drek282** [comment]
They are working fine on the demos, but they are tricky to get working.

**Comment by Drek282** [comment]
This is probably my favourite feature of PS, but creating the heatmap overlays and setting up the proper dimensions, and getting them aligned properly, isn't simple.  There is a spreadsheet called "Heatmap Dimension Checker", in the mod "addons\heatmaps" folder that is intended to help with getting the dimensions and orientation correct.  It has been a while since I've used it so I don't know how well it works.

**Comment by Solomenka** [comment]
But I know hot it works and the problem was in SQL server mode "ONLY_FULL_GROUP_BY". Now I have heatmaps working again.

**Comment by Drek282** [comment]
Great, that's not likely something I would have been able to figure out...

**Comment by Solomenka** [comment]
<OFFTOP>This is your site displaced[dot]zone?</OFFTOP>

**Comment by Drek282** [comment]
Yes, that's the site where the stat demos are hosted.

**Comment by Solomenka** [comment]
There is UA flag at icon, are you from Ukraine?

**Comment by Drek282** [comment]
No, I just hate fascist bullies that attack other nations without just cause, and I think that Vladimir Putin is the greatest danger to world peace and stability since the end of WWII.  If I was younger, and had the skills, the means and the opportunity, I would have gone to the Ukraine and joined the fight against Vladimir Putin.

**Comment by Solomenka** [comment]
Thank you for your support.

P.S. I'm from Ukraine.

**Comment by Drek282** [comment]
Thank you for fighting a brutal dictator that no one else was willing to fight.  You could have chosen to simply surrender, but you did not do that.

**Comment by Solomenka** [comment]
Half-Life Crossfire 24h hourly heatmap 😁

![Image](https://github.com/user-attachments/assets/75c30763-951e-445b-88c1-c79634a67308)

**Comment by Drek282** [comment]
Beautiful.  Those heatmaps were always my favourite PS feature.
